web/book/src/SUMMARY.md

```diff
@@ -60,6 +58,11 @@
     - [Window](./reference/stdlib/transforms/window.md)
 
   - [Aggregation functions]()
+  - [Other Functions](./reference/stdlib/other-functions/README.md)
+    - [Date functions](./reference/stdlib/other-functions/date.md)
+      - [Date & time format](./reference/stdlib/other-functions/date-time-format-specifiers.md)
+    - [Mathematical functions](./reference/stdlib/other-functions/math.md)
+    - [Text functions](./reference/stdlib/other-functions/string.md)
```

Comment:
No particularly strong view, but I'm not sure we need an "Other" category, we could instead promote those three one level?

Reply:
I put them at a higher level. LMK if that's better